src/js/actions/AppActions.js

```diff
@@ -3,146 +3,145 @@ import app    from '../lib/app';
-        ipcRenderer.on('playerAction', (event, reply) => {
-
-            switch(reply) {
-                case 'play':
-                    AppActions.player.play();
-                    break;
-                case 'pause':
-                    AppActions.player.pause();
-                    break;
-                case 'prev':
-                    AppActions.player.previous();
-                    break;
-                case 'next':
-                    AppActions.player.next();
-                    break;
-            }
-        });
-
-        // Prevent some events
-        window.addEventListener('dragover', (e) => {
-            e.preventDefault();
-        }, false);
-
-        window.addEventListener('drop', (e) => {
-            e.preventDefault();
-        }, false);
-
-        // Remember dimensions and positionning
-        const currentWindow = app.browserWindows.main;
-
-        currentWindow.on('resize', () => {
-            AppActions.app.saveBounds();
-        });
-
-        currentWindow.on('move', () => {
-            AppActions.app.saveBounds();
-        });
-    },
-
-    app: {
-
-        start: function() {
-            ipcRenderer.send('appReady');
-        },
-
-        restart: function() {
-            ipcRenderer.send('appRestart');
-        },
-
-        close: function() {
-            app.browserWindows.main.close();
-        },
-
-        minimize: function() {
-            app.browserWindows.main.minimize();
-        },
-
-        maximize: function() {
-            app.browserWindows.main.isMaximized() ? app.browserWindows.main.unmaximize() : app.browserWindows.main.maximize();
-        },
-
-        saveBounds: function() {
-
-            const self = AppActions;
-            const now = window.performance.now();
-
-            if (now - self.lastFilterSearch < 250) {
-                clearTimeout(self.filterSearchTimeOut);
-            }
-
-            self.lastFilterSearch = now;
-
-            self.filterSearchTimeOut = setTimeout(() => {
-
-                app.config.set('bounds', app.browserWindows.main.getBounds());
-                app.config.saveSync();
-
-            }, 250);
-        },
-
-        initShortcuts: function() {
-
-            // Global shortcuts - Player
-            globalShortcut.register('MediaPlayPause', () => {
-                AppActions.player.playToggle();
-            });
-
-            globalShortcut.register('MediaPreviousTrack', () => {
-                AppActions.player.previous();
-            });
-
-            globalShortcut.register('MediaNextTrack', () => {
-                AppActions.player.next();
-            });
-        }
+    window.addEventListener('drop', (e) => {
+        e.preventDefault();
+    }, false);
+
+    // Remember dimensions and positionning
+    const currentWindow = app.browserWindows.main;
+
+    currentWindow.on('resize', () => {
+        saveBounds();
+    });
+
+    currentWindow.on('move', () => {
+        saveBounds();
+    });
```

Comment:
AppAction members are used in  the route file. Like [this](https://github.com/KeitIG/museeks/blob/master/src/js/router/routes.js#L29). How do we handle that?